clusters/nerc-ocp-prod/kustomization.yaml

```diff
@@ -62,3 +64,8 @@ patches:
       - op: replace
         path: /spec/source/path
         value: overlays/nerc-ocp-prod
+      name: gatekeeper
+    patch: |
+      - op: replace
+        path: /spec/source/path
+        value: gatekeeper-system/overlays/nerc-ocp-prod
```

Comment:
```suggestion
  
  - target:
      kind: Application
      name: gatekeeper
    patch: |
      - op: replace
        path: /spec/source/path
        value: gatekeeper-system/overlays/nerc-ocp-prod
```

Reply:
Thank you again!

(a) obviously I am not paying enough attention to what I am typing today, and (b) we need to get our standard manifest validation scripts running on this repository.

Reply:
What repo are those validation scripts in? I don't think I've seen them

Reply:
Our common CI scripts are in https://github.com/OCP-on-NERC/workflows/.

You can see the manifest validation script in use e.g. in https://github.com/OCP-on-NERC/ipmi-exporter/blob/main/.github/workflows/validate-manifests.yaml.

That won't work "out of the box" with this repository because the layout of this repo doesn't match the expectations of the script. Pondering about either updating the script or reorganizing this repository.